src/renderer/components/+extensions/attempt-install-by-info/attempt-install-by-info.tsx

```diff
@@ -109,7 +124,13 @@ export const attemptInstallByInfo = ({ attemptInstall, getBaseRegistryUrl, exten
 
   const url = json.versions[version].dist.tarball;
   const fileName = path.basename(url);
-  const { promise: dataP } = downloadFile({ url, timeout: 10 * 60 * 1000 });
+  const request = await fetch(url, { timeout: 10 * 60 * 1000 });
```

Comment:
`downloadFile` is closer to abstraction level that we should have here. Problem here of course is that everything else is in low level of abstraction making code very hard to read. Guiding factor should be that function should only contain code that is in same level of abstraction thus easier to read.

Reply:
okay I will make it so that fetch is hierarchy of injectables instead.

Reply:
Done